lib/msal-browser/src/event/EventType.ts

```diff
@@ -15,6 +15,7 @@ export enum EventType {
     SSO_SILENT_SUCCESS = "msal:ssoSilentSuccess",
     SSO_SILENT_FAILURE = "msal:ssoSilentFailure",
     HANDLE_REDIRECT_START = "msal:handleRedirectStart",
+    HANDLE_REDIRECT_END = "msal:handleRedirectEnd",
```

Comment:
Do we want to do success and failure?

Reply:
My feeling was that success and failure are covered by the Login and AcquireToken success and failure events